xblock/__init__.py

```diff
@@ -22,4 +22,4 @@ def __init__(self, *args, **kwargs):
 
 xblock.fields.XBlockMixin = XBlockMixin
 
-__version__ = "0.4.7"
+__version__ = "0.5.0"
```

Comment:
Nit: It'd be nice to pull the `setup.py` version and this one from a common location, particularly considering that this version was out-of-date.